src/fastd.c

```diff
@@ -256,11 +266,12 @@ static inline void write_pid(void) {
 		pr_error_errno("can't write PID file: open");
 		goto end;
 	}
+	FILE *f = fdopen(fd, "w");
```

Comment:
I just want to make sure the `0666` mode is preserved as is. Is it ok just to drop it and use `fopen` instead?